openrouteservice/src/main/java/heigit/ors/services/matrix/MatrixServiceSettings.java

```diff
@@ -30,12 +30,12 @@
 		String value = AppConfig.Global().getServiceParameter("matrix", "enabled");
 		if (value != null)
 			enabled = Boolean.parseBoolean(value);		
-		value = AppConfig.Global().getServiceParameter("matrix", "maximum_locations");
+		value = AppConfig.Global().getServiceParameter("matrix", "maximum_routes");
 		if (value != null)
-			maximumLocations = Math.max(1, Integer.parseInt(value));
+			maximumRoutes = Math.max(1, Integer.parseInt(value));
 		value = AppConfig.Global().getServiceParameter("matrix", "maximum_locations_flexible");
```

Comment:
This should be updated to "maximum_routes_flexible" to match the "maximum_routes" attibute